src/axom/sidre/core/Group.cpp

```diff
@@ -2281,6 +2281,38 @@ bool Group::loadExternalData(const hid_t& h5_id)
   return !(getDataStore()->getConduitErrorOccurred());
 }
 
+/*
+ *************************************************************************
+ *
+ * Load External Data from a path within an hdf5 file
```

Comment:
Should this say Sidre group path instead? Could there be confusion about whether the path refers to a Sidre path or an HDF5 path?

Also, should the `hid_t` arg be documented like it is further down in the file (i.e., "HDF5 handle")? Same comment for the method above?